monitor/dbt_project/macros/get_new_model_alerts.sql

```diff
@@ -1,19 +1,24 @@
 {% macro get_new_model_alerts(days_back) %}
     -- depends_on: {{ ref('alerts_models') }}
     {% set select_new_alerts_query %}
-        WITH alerts AS (
-            SELECT * FROM {{ ref('alerts_models') }}
-            WHERE alert_sent = FALSE and detected_at >= {{ get_alerts_time_limit(days_back) }}
+        with alerts as (
+            select * from {{ ref('alerts_models') }}
+            where alert_sent = false and detected_at >= {{ get_alerts_time_limit(days_back) }}
         ),
-
-        models AS (
-            SELECT * FROM {{ ref('elementary', 'dbt_models') }}
+        models as (
+            select * from {{ ref('elementary', 'dbt_models') }}
+        ),
+        snapshots as (
+            select * from {{ ref('elementary', 'dbt_snapshots') }}
         )
 
-        SELECT
+        select
             alerts.*,
-            models.meta as meta
-        FROM alerts LEFT JOIN models on (alerts.unique_id = models.unique_id)
+            models.meta as model_meta,
+            snapshots.meta as snapshot_meta
```

Comment:
I think that we don't need two separate fields here because if you can't have both model_meta and shanpshot_meta. I'm suggesting having one meta field that is either the snapshots.meta or the models.meta

Reply:
How would you do it at the SQL-level?
Though, I can change:
```py
                                 'model_meta': elementary.insensitive_get_dict_value(model_result_alert_dict, 'model_meta'),
                                 'snapshot_meta': elementary.insensitive_get_dict_value(model_result_alert_dict, 'snapshot_meta'),
```
To:
```
'meta': elementary.insensitive_get_dict_value(model_result_alert_dict, 'model_meta') or elementary.insensitive_get_dict_value(model_result_alert_dict, 'snapshot_meta'),
```